File: AppSets-iOS/AppSets/com/app/appsets/im/RabbitMQBrokerProperty.swift
//
//  RabbitMqBrokerProperty.swift
//  AppSets
//
//  Created by caiju Xu on 2024/7/9.
//

import Foundation

struct RabbitMqBrokerProperty: Codable {
    let uid: String
    let userExchangeGroups:String?
    let host:String
    let port:Int
    let username:String
    let password:String
    let virtualHost:String
    let queuePrefix:String
    let routingKeyPrefix:String
    let groupExchangePrefix:String
    let groupExchangeParent:String
    let groupRootExchange:String
    let groupSubRootExchange:String
}

File: AppSets-Android/appsets/src/main/kotlin/xcj/app/appsets/im/RabbitMQBrokerProperty.kt
package xcj.app.appsets.im

data class RabbitMQBrokerProperty(
    val host: String,
    val port: Int,
    val username: String,
    val password: String,
    val virtualHost: String,
    val queuePrefix: String,
    val routingKeyPrefix: String,
    val groupExchangePrefix: String,
    val groupExchangeParent: String,
    val groupRootExchange: String,
    val groupSubRootExchange: String,
)